### ISSUE TITLE
[Crash]: Rendreing Entity in World Straw Status Skin crash player

### ISSUE THREAD

**Issue by Arginou57** [Fuzss/strawstatues#43]
### Mod Loader (Required)

Forge

### Minecraft Version(s) (Required)

1.20.1

### Mod Version(s) (Required)

v8.0.3

### Minimal Setup

No

### Notes (Required)

![image](https://github.com/user-attachments/assets/778ff6f6-9297-4884-8c84-c141e3c6048e)
![image](https://github.com/user-attachments/assets/ede6d12c-7126-46a7-8400-fb8ddc50a2ca)

The error appear when using Militaire Tag in the Skin StrawStatus

### Crash Report (Required)

https://hastebin.com/share/qamipebewo.perl

**Comment by Aspurgus** [comment]
ive been having this issue as well, the problem only seems to be with the cape layer. If the cape is toggled off, everything is fine, and player names who dont have a cape render fine too.

**Comment by SandwichKnight158** [comment]
Yeah I had this bug too and it nearly corrupted my server. Thankfully uninstalling removed the entity, but I think removing the cape option would be useful for now

**Comment by Fuzss** [comment]
You have to post logs and crash reports guys, otherwise there is nothing that can be done.

**Comment by SandwichKnight158** [comment]
> You have to post logs and crash reports guys, otherwise there is nothing that can be done.

[crash-2025-03-09_00.05.00-client.txt](https://github.com/user-attachments/files/19147922/crash-2025-03-09_00.05.00-client.txt)

**Comment by Fuzss** [comment]
That's a bug with Entity Model Features, please report it to them.
